package.json

```diff
@@ -1,73 +1,27 @@
 {
-  "homepage": "http://LiangJoshua.github.io/calisthantics-web",
-  "name": "calisthantics-web",
+  "name": "cali",
```

Comment:
Reason behind changes in package and package-lock? Did this come from yarn install?

Reply:
Yes I didn't edit that part in the file

Reply:
Can you revert your changes and try everything with npm install of yarn, and then test that everything still works? It's normal to have things added to package or dependency versions to change, but we definitely don't want dependencies being removed. I can do this with you if it gets confusing. If you aren't familiar with package, package and package-lock are two really sensitive files that hold all of the 3rd party libraries/packages that we'll be using